run.sh

```diff
@@ -15,11 +15,11 @@ export KEYPRESS="xvkbd -xsendevent -secure -text"
 export TERMINAL="tmux new-window "
 
 # Use system speech synthesizer on macOS
-if [[ "uname" == "Darwin" ]]
+if [ "$(whoami)" != 'Darwin' ]
```

Comment:
Also, probably better to do a positive match with '=', so platform-specific settings are more directly obvious..